src/ensemble.jl

```diff
@@ -0,0 +1,13 @@
+struct EnsembleOptimizationProblem{T1} <: SciMLBase.AbstractEnsembleProblem
+    prob::OptimizationProblem{iip, F, T1} where {iip, F}
+    u0s::Vector{T1}
+end
+
+function SciMLBase.__init(prob::EnsembleOptimizationProblem{T}, args...; kwargs...) where {T <: OptimizationProblem}
```

Comment:
why is this not just using the standard EnsembleProblem? There's no reason for that to be different?

Reply:
It was my intention to use it but I couldn't see how to use that here since that's built around handling timeseries outputs, I might be wrong though

Reply:
It doesn't require timeseries? NonlinearProblem does it without time series.

Reply:
We want that to be the ensemble interface for all SciMLProblems.